### PR TITLE
Minor fixes: slack message consistency, docker command shorthand, non-anon db volume

### PR DESCRIPTION
**Pull request by jaybuidl**
- Slack message consistency
- Docker command shorthand for testing
- Non-anon db volume (otherwise it does't get deleted by `docker compose down -v`)

## Summary by CodeRabbit

- **New Features**
	- Added scripts for managing Docker containers in the staging environment.
	- Introduced commands to start and stop Docker containers using a staging environment configuration.

- **Improvements**
	- Enhanced data persistence for the database service with new volume configurations.

### CODE REVIEW

**Comment by fcanela** [comment]
@jaybuidl Can this PR be splitted in parts? I absolutely want to merge the Slack correction immediately, but I would prefer to discuss the package.json and docker related changes separately.

File: package.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "bot:run": "node dist/start.js",
     "test:all": "mocha",
     "test:coverage": "c8 --reporter html --reporter text --all --src ./src mocha",
-    "test": "yarn test:coverage"
+    "test": "yarn test:coverage",
+    "docker:up:staging": "docker compose --env-file .env.staging up --build bot-dev",
```
Comment (fcanela) [inline package.json:16]: I would prefer not to implement this pattern. In the application, the runner is docker and npm has environment-agnostic tasks. With the change, we have npm running docker to run npm again, making not so evident the execution flow. Also the `.env.staging` file does not exist in other environments.

File: templates/slack/answer-issued.ejs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-*`<%- notification.space.ens %>`: 💰 New Reality Answer!*
+*<%- notification.space.ens %>: 💰 New Reality Answer!*
```
Comment (fcanela) [inline templates/slack/answer-issued.ejs:1]: I would love to merge this ASAP so it is ready in the next deployment. Do you mind issuing a different PR for that? (I can replicate the changes if that makes it easier)

File: package.json (hunk 1 above)
Comment (jaybuidl) [reply package.json:16]: Ok, moved to shell scripts

**Pull request merged**